### PR TITLE
Add query to check for "favorite" food items

### PR DESCRIPTION
**Pull request by raahimenon**
## Overview

Frontend can pass in a list of "favorites" as strings; any food in the list will be marked as a favorite in the response body.

## Changes Made
Favorites were added to the resolver for campus eateries and eateries (which both link to the same function). To use this, frontend can add a `(favorites: [LIST OF STRINGS])` to their query after `eateries` or `campusEateries`. 

## Test Coverage
Manual testing.

## Next Steps (delete if not applicable)
Add a route to store favorites along with netIDs for users that are signed in to the app.

### CODE REVIEW

File: src/gql_parser/campus_eatery.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 )
 
 
-def get_campus_eateries(eatery_id):
+def get_campus_eateries(eatery_id, favorites=[]):
```
[the remark this answers is not in the document]
Comment (grebneerg) [reply src/gql_parser/campus_eatery.py]: why is python like this

**Review by grebneerg — CHANGES_REQUESTED** [review]
Once you address hound's comments this should be good.

File: src/schema.py [hunk 2]
```diff
@@ -33,18 +33,18 @@ def update_collegetown_data(collegetown_eateries):
...
-        return get_campus_eateries(eatery_id)
+    def resolve_campus_eateries(self, info, eatery_id=None, favorites=[]):
+        return get_campus_eateries(eatery_id, favorites)
 
     def resolve_collegetown_eateries(self, info, eatery_id=None):
         return get_collegetown_eateries(eatery_id)
 
-    def resolve_eateries(self, info, eatery_id=None):
-        return get_campus_eateries(eatery_id)
+    def resolve_eateries(self, info, eatery_id=None, favorites=[]):
```
[the remark this answers is not in the document]
Comment (alanna-zhou) [reply src/schema.py]: lol hound u need to calm down

**Review by alanna-zhou — APPROVED** [review]
hound needs to calm down but u are very cool thanku

**Review by grebneerg — APPROVED** [review]
Looks good

**Pull request merged**